storage/etcd/config.go

```diff
@@ -0,0 +1,82 @@
+package etcd
+
+import (
+	"time"
+
+	"github.com/coreos/dex/storage"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/coreos/etcd/pkg/transport"
+	"github.com/sirupsen/logrus"
+)
+
+type EtcdSSL struct {
+	ServerName string `json:"server_name" yaml:"server_name"`
+	CAFile     string `json:"ca_file" yaml:"ca_file"`
+	KeyFile    string `json:"key_file" yaml:"key_file"`
+	CertFile   string `json:"cert_file" yaml:"cert_file"`
+}
+
+type Etcd struct {
+	Endpoints        []string `json:"endpoints" yaml:"endpoints"`
+	DialTimeout      int      `json:"dial_timeout" yaml:"dial_timeout"`
```

Comment:
We generally don't make everything configurable just because the client library does. Can we set DialTimeout and AutoSyncInterval to reasonable defaults and if there are compelling reasons to expose them later we can?

Reply:
Hmm that works for me to. I can set them to the same default as etcdctl.